src/main/python/deep_qa/training/trainer.py

```diff
@@ -518,6 +573,7 @@ def _save_best_model(self):
         epoch_weight_file = "%s_weights_epoch=%d.h5" % (self.model_prefix, self.best_epoch)
         final_weight_file = "%s_weights.h5" % self.model_prefix
         copyfile(epoch_weight_file, final_weight_file)
+        print("Saved the best model to {}".format(final_weight_file))
```

Comment:
does logger get suppressed? it didn't come up when i was trying to run the test.